vision/blob/blob.py

```diff
@@ -0,0 +1,53 @@
+from vision.blob.object_type import ObjectType
+
+class Rectangle:
+    """
+    Constructs a 3D rectangular prism using 8 (x, y, z) coordinates
+
+    Parameters
+    ----------
+    top_left_near: tuple[float]
+        top (-y), left (-x), and near (-z) coordinate
+    top_right_near: tuple[float]
+        top (-y), right (+x), and near (-z) coordinate
+    bottom_right_near: tuple[float]
+        bottom (+y), right (+x), and near (-z) coordinate
+    bottom_left_near: tuple[float]
+        bottom (+y), left (-x), and near (-z) coordinate
+    top_left_far: tuple[float]
+        top (-y), left (-x), and far (+z) coordinate
+    top_right_far: tuple[float]
+        top (-y), right (+x), and far (+z) coordinate
+    bottom_right_far: tuple[float]
+        bottom (+y), right (+x), and far (+z) coordinate
+    bottom_left_far: tuple[float]
+        bottom (+y), left (-x), and far (+z) coordinate
+    """
+    def __init__(self, top_left_near, top_right_near, bottom_right_near, bottom_left_near, top_left_far, top_right_far, bottom_right_far, bottom_left_far):
+        self.top_left_near = top_left_near
+        self.top_right_near = top_right_near
+        self.bottom_right_near = bottom_right_near
+        self.bottom_left_near = bottom_left_near
+        self.top_left_far = top_left_far
+        self.top_right_far = top_right_far
+        self.bottom_right_far = bottom_right_far
+        self.bottom_left_far = bottom_left_far
+
+class Blob:
```

Comment:
I think you should add a object_type parameter to rectangle rather than creating a wrapper.